lighthouse-core/test/lib/stack-packs-test.js

```diff
@@ -120,4 +127,23 @@ describe('stack packs', () => {
     `);
     /* eslint-enable max-len */
   });
+
+  it('snapshot unrecognized keys', () => {
+    const auditIds = getAuditIds();
+
+    const unrecognizedKeys = new Set();
+    for (const pack of lighthouseStackPacks) {
+      for (const key in pack.UIStrings) {
+        if (!auditIds.includes(key)) unrecognizedKeys.add(key);
+      }
+    }
+
+    expect([...unrecognizedKeys]).toMatchInlineSnapshot(`
+      Array [
+        "time-to-first-byte",
```

Comment:
huzzah catching bugs already! I'm guessing it should be `server-response-time` since it was renamed from `time-to-first-byte` :)

Reply:
cool, but punt https://github.com/GoogleChrome/lighthouse-stack-packs/issues/50

Reply:
sure punt is fine, wasn't trying to flag this as a blocker, just celebrating the success of its implementation 🎉  :)